src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/haflow/create/actions/CompleteHaFlowCreateAction.java

```diff
@@ -106,7 +106,11 @@ protected void perform(
                 subFlow.setStatus(subFlowMap.getOrDefault(subFlow.getHaSubFlowId(), FlowStatus.UP));
             }
             haFlow.setStatus(haFlowStatus);
-            //TODO add status info?
+            if (FlowStatus.DEGRADED.equals(haFlowStatus)) {
+                haFlow.setStatusInfo("An alternative way (back up strategy or max_latency_tier2 value) of building "
```

Comment:
shouldn't it be a constant somewhere? Or the info is used only in this single place?

Reply:
I created a constant for it